src/lib.rs

```diff
@@ -10,13 +10,22 @@ use std::path::Path;
 use std::ops::Drop;
 use std::ptr;
 use std::mem::forget;
-use libc::free;
+use std::slice;
 use deepspeech_sys as ds;
 
 pub struct Model {
 	model :* mut ds::ModelState,
 }
 
+pub struct Metadata {
+	metadata :* mut ds::Metadata,
+}
+
+#[repr(transparent)]
+pub struct MetadataItem {
+	metadata_item : ds::MetadataItem,
```

Comment:
Could you do `metadata_item :ds::MetadataItem`? My code style has `a :b` when b is a type, and `a : b` when b is a value that `a` is assigned to.